localstack/services/apigateway/integration.py

```diff
@@ -896,3 +896,31 @@ def apply_request_parameters(
                 query_params.pop(key)
 
     return add_query_params_to_url(uri, query_params)
+
+
+class EventBridgeIntegration(BackendIntegration):
+    def invoke(self, invocation_context: ApiInvocationContext):
+        invocation_context.context = get_event_request_context(invocation_context)
+        try:
+            payload = self.request_templates.render(invocation_context)
+        except Exception as e:
+            LOG.warning("Failed to apply template for EventBridge integration: %s", e)
+            raise
+        uri = (
+            invocation_context.integration.get("uri")
+            or invocation_context.integration.get("integrationUri")
+            or ""
+        )
+        region_name = uri.split(":")[3]
+        headers = aws_stack.mock_aws_request_headers(service="events", region_name=region_name)
```

Comment:
Can we use `get_internal_mocked_headers` here as well? It adds internal headers which are necessary for IAM policy enforcement.